activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -280,6 +280,26 @@ class TooManyRecords < ActiveRecordError
     #   member = Member.new
     #   member.avatar_attributes = {icon: 'sad'}
     #   member.avatar.width # => 200
+    #
+    # === Forms
+    #
+    # Use ActionView::Helpers::FormHelper::fields_for to create form elements
+    # for updating or destroying nested attributes.
+    #
+    # === Testing
+    #
+    # If you are using ActionView::Helpers::FormHelper::fields_for, your integration
+    # tests should replicate the HTML structure it provides. For example;
+    #
+    #   post members_path, params: {
+    #     member: {
+    #       name: 'joe',
+    #       posts_attributes: {
+    #         "0" => { title: 'Foo' },
+    #         "1" => { title: 'Bar' }
```

Comment:
```suggestion
    #       name: "joe",
    #       posts_attributes: {
    #         "0" => { title: "Foo" },
    #         "1" => { title: "Bar" }
```

Reply:
I prefer double quotes, but the rest of this doc uses single quotes. See https://api.rubyonrails.org/classes/ActiveRecord/NestedAttributes/ClassMethods.html

Reply:
Good point, I was mostly interested in consistency (either the ones I suggested or the `"0"` and `"1"`)

Reply:
Oh yeah of course... I'll fix the 0 and 1.